src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -114,6 +116,28 @@ private void configChange() {
         OpenShiftUtils.initializeOpenShiftClient(server);
         this.namespace = getNamespaceOrUseDefault(namespace, getOpenShiftClient());
 
+        logger.info("Waiting for Jenkins to be started");
```

Comment:
Good question. TBH I'm not 100% sure that everything is in the right state when `ItemListener.onLoaded` is called. The current way may not be the "best" or most correct way to do it but we know it works. I've raised fabric8io/openshift-jenkins-sync-plugin#109 to track it & test it out once I get some time.